example/android/app/src/main/java/com/adobe/marketing/mobile/flutter/flutter_aepsdk_example/MyApplication.java

```diff
@@ -26,7 +26,7 @@ public void onCreate() {
             Lifecycle.registerExtension();
             Signal.registerExtension();
             Assurance.registerExtension();
-            Consent.registerExtension();
```

Comment:
Why do we need to provide the full package path for consent here?

Reply:
It is because we import com.adobe.marketing.mobile.*; only.  And it can't recognize Consent.registerExtension() until I use the full path.

Reply:
Let's import consent and then use it.

`import com.adobe.marketing.mobile.edge.consent.Consent`

Reply:
Let bring @emdobrin and @cdhoffmann in.
Because we took out the import purposely from the earlier PR.
So in the Sample App we have 
import com.adobe.marketing.mobile.*; 

But consent will need to be register as 
com.adobe.marketing.mobile.edge.consent.Consent.registerExtension();

Should we add back
import com.adobe.marketing.mobile.edge.consent.Consent
and register as Consent.registerExtension();  ?

I am fine with either way.

Reply:
Updated, thanks @addb @emdobrin.